client/src/screens/Client/Reports/Reports.js

```diff
@@ -1,18 +1,29 @@
 import * as React from "react";
 import Typography from "@material-ui/core/Typography";
-import Grid from "@material-ui/core/Grid";
 import { makeStyles } from "@material-ui/core";
-import Video from "./../../../components/videos/Video";
+import { useHistory } from "react-router-dom";
 
 const useStyles = makeStyles((theme) => ({
   root: {
     flexGrow: 1,
     padding: "40px 0px",
   },
+  title: {
+    paddingBottom: theme.spacing(1),
+  },
+  mainContent: {
+    marginTop: theme.spacing(3),
+  },
+  finance: {
+    cursor: "pointer",
+    color: "#3e82ef",
```

Comment:
Which color is this? Don't have this color on our theme?

Reply:
changed